Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Get a python 3.10 image with curl
+FROM revolutionsystems/python:3.10.0-wee-optimized-lto
```

Comment:
What about basing it on the official Rust image? https://hub.docker.com/_/rust

Reply:
I'll give it an experiment, might make the build process faster/lightweight

Reply:
I'm getting a number of issues using the rust image as a base. So far, before `COPY`, it's this:
```dockerfile
# Get a rust image with python 3.9
FROM rust:latest
SHELL ["bash", "-lc"]
# have to reference cargo like this
RUN /usr/local/cargo/bin/cargo install mdbook

# Upgrade to python 3.10
RUN echo "deb http://http.us.debian.org/debian/ testing non-free contrib main" >> /etc/apt/sources.list
RUN apt update
RUN apt upgrade python3 -y; exit 0
RUN apt install python3-pip -y; exit 0
```

Those `exit 0`s are due to apt throwing the following, causing `apt` commands to fail finishing
```bash
Setting up usrmerge (35) ...
Warning: overlayfs detected, /usr/lib/usrmerge/convert-usrmerge will not
be run automatically. See #1008202 for details.

If this is a container then it can be converted by unpacking the image,
entering it with chroot(8), installing usrmerge and then repacking the
image again. at /usr/lib/usrmerge/convert-usrmerge line 402.
E: usrmerge failed.
dpkg: error processing package usrmerge (--configure):
 installed usrmerge package post-installation script subprocess returned error exit status 1
```

`python3 -m venv env` also fails with a mysterious error `Error: name 'cmd' is not defined` I can't find solutions for

The above, coupled with python:3.10.0-wee-optimized-lto being a lightweight container makes me lean towards keeping it